### ISSUE TITLE
Train model on custom dataset from file does not work

### ISSUE THREAD

**Issue by DavidHerel** [QData/TextAttack#625]
**Describe the bug**
Train model on custom dataset returns failure.

**To Reproduce**
1. Run following command `!textattack train --model distilbert-base-uncased --dataset-from-file <*.py> --model-num-labels 2 --model-max-length 64 --per-device-train-batch-size 128 --num-epochs 3`

**Expected behavior**
Expected to train the model from the custom dataset

**Screenshots or Traceback**
![image](https://user-images.githubusercontent.com/49339671/157267783-82bd1249-aced-4ea6-8e0c-80db302c54dc.png)

**System Information (please complete the following information):**
 - OS: Windows
 - Textattack version: latest

**Comment by qiyanjun** [comment]
@VijayKalmath was right — \`train\` genuinely didn't support \`--dataset-from-file\` at all; unlike \`attack\`/\`eval\`, its own arg parser (\`_CommandLineTrainingArgs\`) never registered the flag or wired it into dataset construction. This wasn't a bug in existing code, it was a missing feature.

Implemented it in #832: \`--dataset-from-file <path>\` (optionally \`path^module_prefix\`) now works for \`train\`, loading a Python module that defines \`train_dataset\` and \`eval_dataset\` (each a \`textattack.datasets.Dataset\`). Verified end-to-end with a fake module. Will close automatically once that merges.
